packages/mui-lab/src/DateRangePickerDay/DateRangePickerDay.tsx

```diff
@@ -41,7 +41,8 @@ export interface DateRangePickerDayClasses {
 
 export type DateRangePickerDayClassKey = keyof DateRangePickerDayClasses;
 
-export interface DateRangePickerDayProps<TDate> extends Omit<PickersDayProps<TDate>, 'classes'> {
+export interface DateRangePickerDayProps<TDate = unknown>
```

Comment:
Looking at `packages/mui-lab/src/themeAugmentation/props.d.ts`. Shouldn't it be like this instead?

```ts
export interface LabComponentsPropsList {
  MuiAvatarGroup: AvatarGroupProps;
  MuiCalendarPicker: CalendarPickerProps<unknown>;
  MuiCalendarPickerSkeleton: CalendarPickerSkeletonProps;
  MuiClockPicker: ClockPickerProps<unknown>;
  MuiDatePicker: DatePickerProps;
  MuiDateRangePickerDay: DateRangePickerDayProps<unknown>;
```

Reply:
If we do this `MuiDateRangePickerDay: DateRangePickerDayProps<unknown>;`, 

do we need to set the default value here? `export interface DateRangePickerDayProps<TDate = unknown>`. 

I think we only need either? Correct me if I am wrong.

Reply:
I don't think the default type is needed.